includes/payment-methods/class-affirm-payment-method.php

```diff
@@ -24,12 +25,13 @@ class Affirm_Payment_Method extends UPE_Payment_Method {
 	 */
 	public function __construct( $token_service ) {
 		parent::__construct( $token_service );
-		$this->stripe_id           = self::PAYMENT_METHOD_STRIPE_ID;
-		$this->title               = __( 'Affirm', 'woocommerce-payments' );
-		$this->is_reusable         = false;
-		$this->icon_url            = plugins_url( 'assets/images/payment-methods/affirm.svg', WCPAY_PLUGIN_FILE );
-		$this->currencies          = [ 'USD', 'CAD' ];
-		$this->limits_per_currency = [
+		$this->stripe_id                     = self::PAYMENT_METHOD_STRIPE_ID;
+		$this->title                         = __( 'Affirm', 'woocommerce-payments' );
+		$this->is_reusable                   = false;
+		$this->icon_url                      = plugins_url( 'assets/images/payment-methods/affirm.svg', WCPAY_PLUGIN_FILE );
+		$this->currencies                    = [ 'USD', 'CAD' ];
+		$this->accept_only_domestic_currency = true;
+		$this->limits_per_currency           = [
```

Comment:
I added some comments asking to tweak naming `limits_per_currency` -> `limits_per_buyer_currency` and `countries` -> `buyer_countries`, but don't see them. Duplicating the request once more.

Reply:
> I added some comments asking to tweak naming limits_per_currency -> limits_per_buyer_currency and countries -> buyer_countries, but don't see them.

Sorry, I can't find any comment about this request. 

Can you elaborate the reasoning for this change? I wonder why not also `currencies` -> `buyer_currencies` then?

Reply:
Sure: it's to align with naming we have on the server side (see transactions cache tables, parameters naming for transactions endpoints). Having the naming aligned makes it easier to draw connections.

Reply:
Sorry, I can't see how it makes things better. I don't see any reference of `buyer_...` either in associated cache class or endpoints. We have `customer_currency` and `customer_country` in the cache, for example.

Also, I don't see how it's related to this PR. Can we maybe create a separate PR and move any related discussion there? I'm happy to create the PR and take it on so that it's not lost in prioritisation.

Reply:
You are right about naming, it's `customer_` in DB.